app/components/shared/inputs/TwitchTagsInput.vue.ts

```diff
@@ -0,0 +1,55 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import VSelectPage from 'v-selectpage';
+import { Inject } from 'util/injector';
+import { $t, I18nService } from 'services/i18n';
+import { prepareOptions, TwitchTag, TwitchTagWithLabel } from 'services/platforms/twitch/tags';
+
+Vue.use(VSelectPage);
+
+@Component({})
+export default class TwitchTagsInput extends Vue {
```

Comment:
Components inside `shared/inputs` must be consistent with other components.  It must extend `BaseInput` and have all props beside `value` in the `metadata` property. For example, you can take a look on `TextInput`.

Reply:
If you don't want your component to be consistent with other Inputs please rename it to something like `TwitchTagsEditor` and place outside the `shared/inputs` folder

Reply:
Work on this, including extracting/generalizing components is now on a separate branch. I don't think we should include on this one.